### ISSUE TITLE
Not asking for pin when deleting wallet

### ISSUE THREAD

**Issue by chasingkirkjufell** [navcoin/WhisperWallet#112]
When clicking on deleting the wallet, users should be asked to confirm and then re-enter the pin. I didn't check the biometric lock, but in the biometric case, user should be ask to confirm the action first and then check for biometric to avoid accident clicks and conformation.

**Comment by aguycalled** [comment]
I think it should also ask for explicit confirmation:

```
The coins stored on the wallet will be lost and only accessible again using a valid backup. Please be sure your seed words are correctly backed up.

Are you sure you want to delete this wallet?
```

**Comment by aguycalled** [comment]
fixed in 6ee4962
